[2023-11-12T21:09:13.694+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=simulator_dag_11/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_11 permission to 509
[2023-11-12T21:09:13.698+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=simulator_dag_11/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_11 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=simulator_dag_11/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_11'
[2023-11-12T21:09:13.754+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=simulator_dag_11/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_11 permission to 509
[2023-11-12T21:09:13.756+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=simulator_dag_11/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_11 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=simulator_dag_11/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_11'
[2023-11-12T21:09:13.820+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simulator_dag_11.setup_and_run_simulator_task_11 scheduled__2023-11-11T00:00:00+00:00 [queued]>
[2023-11-12T21:09:13.839+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simulator_dag_11.setup_and_run_simulator_task_11 scheduled__2023-11-11T00:00:00+00:00 [queued]>
[2023-11-12T21:09:13.843+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-11-12T21:09:13.868+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): setup_and_run_simulator_task_11> on 2023-11-11 00:00:00+00:00
[2023-11-12T21:09:13.878+0000] {standard_task_runner.py:57} INFO - Started process 190 to run task
[2023-11-12T21:09:13.884+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simulator_dag_11', 'setup_and_run_simulator_task_11', 'scheduled__2023-11-11T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/timeseries.py', '--cfg-path', '/tmp/tmp9pdohli9']
[2023-11-12T21:09:13.890+0000] {standard_task_runner.py:85} INFO - Job 37: Subtask setup_and_run_simulator_task_11
[2023-11-12T21:09:13.978+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=simulator_dag_11/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_11 permission to 509
[2023-11-12T21:09:13.980+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=simulator_dag_11/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_11 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=simulator_dag_11/run_id=scheduled__2023-11-11T00:00:00+00:00/task_id=setup_and_run_simulator_task_11'
[2023-11-12T21:09:13.985+0000] {task_command.py:416} INFO - Running <TaskInstance: simulator_dag_11.setup_and_run_simulator_task_11 scheduled__2023-11-11T00:00:00+00:00 [running]> on host 84c765240aba
[2023-11-12T21:09:14.115+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='simulator_dag_11' AIRFLOW_CTX_TASK_ID='setup_and_run_simulator_task_11' AIRFLOW_CTX_EXECUTION_DATE='2023-11-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-11T00:00:00+00:00'
[2023-11-12T21:09:14.145+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-12T21:09:14.166+0000] {logging_mixin.py:154} INFO - Simulator process started
[2023-11-12T21:09:14.169+0000] {logging_mixin.py:154} INFO - Simulator process ID: 192
[2023-11-12T21:09:14.172+0000] {logging_mixin.py:154} INFO - Waiting for simulator process to complete...
[2023-11-12T21:09:14.191+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simulator_dag_11, task_id=setup_and_run_simulator_task_11, execution_date=20231111T000000, start_date=20231112T210913, end_date=20231112T210914
[2023-11-12T21:09:14.284+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-12T21:09:14.312+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
